.gitignore

```diff
@@ -0,0 +1,189 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There shouldn't be a .gitignore file in the repository, especially not for build files. I recommend you do out-of-source builds! If you need/want a .gitignore file, add one to the clone directory and add .gitignore in the file, so that it doesn't get committed.